int16[] sensor_values
